ls1intum/Artemis
# Change Instruction Component for File Upload Exercises

### Checklist

### Motivation and Context



Closes #2141

### Description


- Switched to `jhi-resizable-instructions` to `jhi-collapsable-assessment-instructions` for file upload exercises

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Assess a File Upload Exercise and check that the instruction panel works

### Screenshots



![image](https://user-images.githubusercontent.com/29718932/94908580-311b0700-04a2-11eb-81d9-a08b284ed1d9.png)
